Plugin/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "extends": "airbnb-base",
+    "plugins": [
+        "import"
+    ],
+    "rules": {
+        "max-len": "off",
+        "brace-style": ["error", "stroustrup"],
+        "no-unused-vars": ["error", { "args": "none" }]
```

Comment:
Just a note we added a rule in [this](https://github.com/mozillascience/CitationCore/pull/6) PR.  Not merged, but we might want to add that rule here.